app/models/katello/activation_key.rb

```diff
@@ -92,11 +168,11 @@ def related_resources
     end
 
     def available_releases
-      if self.environment
-        self.environment.available_releases
-      else
-        self.organization.library.available_releases
+      releases = self.content_view_environments.flat_map do |cve|
+        cve.content_view.version(cve.lifecycle_environment).available_releases
```

Comment:
Why change the logic here to look at cv instead of lce?

Reply:
was trying to mimic content facet => https://github.com/Katello/katello/blob/master/app/models/katello/host/content_facet.rb#L322-L326